diff --git a/node_modules/@web3-react/walletconnect-connector/dist/walletconnect-connector.esm.js b/node_modules/@web3-react/walletconnect-connector/dist/walletconnect-connector.esm.js
index 5e4aad3..747118c 100644
--- a/node_modules/@web3-react/walletconnect-connector/dist/walletconnect-connector.esm.js
+++ b/node_modules/@web3-react/walletconnect-connector/dist/walletconnect-connector.esm.js
@@ -125,7 +125,6 @@ var WalletConnectConnector = /*#__PURE__*/function (_AbstractConnector) {
         bridge = _ref.bridge,
         qrcode = _ref.qrcode,
         pollingInterval = _ref.pollingInterval;
-    !(Object.keys(rpc).length === 1) ? process.env.NODE_ENV !== "production" ? invariant(false, '@walletconnect/web3-provider is broken with >1 chainId, please use 1') : invariant(false) : void 0;
     _this2 = _AbstractConnector.call(this, {
       supportedChainIds: Object.keys(rpc).map(function (k) {
         return Number(k);
